Jenkinsfile_k8s

```diff
@@ -0,0 +1,27 @@
+def dockerImage = 'jenkinsciinfra/hashicorp-tools:0.3.7' // Tracked by updatecli
```

Comment:
Is it mandatory? I might have missed something but can't updatecli localize the version if it were specified inline L23?

Reply:
There are 2 locations where this  value is used, hence the variable to factorize.